src/carousel/carousel.js

```diff
@@ -219,7 +223,8 @@ angular.module('ui.bootstrap.carousel', [])
     scope: {
       interval: '=',
       noTransition: '=',
-      noPause: '='
+      noPause: '=',
+      noWrap: '=?'
```

Comment:
Since this is readonly, use `'&'` instead and read it using `$scope.noWrap()`. This saves on a watcher.
We'll eventually change the rest of these scope variables to use `'&'` for performance. I'll create another issue for that.